docs/source/release-notes.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: GraphOS Router Release Notes
+---
+
+## GraphOS Router v2.0 (Alpha)
```

Comment:
Will remove this for this PR, if the upgrading-from-version-1.x guide is approved to be sufficient for this release. Will be adding for subsequent releases